spec/db/seeds_spec.rb

```diff
@@ -21,7 +21,8 @@
     allow(Rails.env).to receive(:review?).and_return(true)
   end
 
-  it "sets up correct data" do
+  # Doing this in one test should save ~2 minutes
```

Comment:
If this takes so long to run, would it make sense to create fewer resources in the seed for test env only?

Reply:
Oh, okay, of course